test/packages/multiinput/data_stream/test/agent/stream/tcp.yml.hbs

```diff
@@ -0,0 +1,22 @@
+tcp:
+host: "{{tcp_host}}:{{tcp_port}}"
```

Comment:
Is the indentation here correct?

Reply:
Turns out is not an indentation problem. That `tcp` key is just meaningless. Seems like a few packages include this mistake and we've just been repeating it in some places.

Reply:
Okay, thanks. Can you make an issue (or PR, whatever is easier), probably in the integrations repo to clean up the meaningless keys?